src/py21cmmc/likelihood.py

```diff
@@ -965,3 +965,139 @@ def define_noise(self, ctx, model):
             return [{"sigma": s(m["Muv"])} for s, m in zip(sig, model)]
         else:
             return [{"sigma": s} for s in sig]
+
+
+class LikelihoodEDGES(LikelihoodBaseFile):
+    """
+    A likelihood based on chi^2 comparison to Global Signal of EDGES timing and fwhm
+
+    This is the likelihood arising from Bowman et al. (2018), which reports an absorption feature
+    in the 21-cm brightness temperature spectra
+
+    Parameters
+    ----------
+
+    use_width : bool
+        whether to use the fwhm in the likelihood, by default it's False
+
+    """
+
+    freq_edges = 78.0
+    freq_err_edges = 1.0
+    fwhm_edges = 19.0
+    fwhm_err_upp_edges = 4.0
+    fwhm_err_low_edges = 2.0
+
+    required_cores = [core.CoreLightConeModule]
+
+    def __init__(self, use_width=False, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.use_width = use_width
+
+    def reduce_data(self, ctx):
+        frequencies = 1420.0 / (np.array(ctx.get("lightcone").node_redshifts) + 1)
```

Comment:
This should be 1420.4

I know it won't change any results, but it should at least be accurate. Probably should explicitly be 1.0 not 1, not sure how much is matters in Python for adding different variable types (or if it is intelligent enough to know that its a double)

Reply:
can I confirm with you that it is 1419.4697 or 1420.4?

Reply:
It is 1420.405751...

Reply:
of course... now I'm wondeing why I've been using 1419 since the beginning ...